bash/sqlfluff_dbt.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+command=$1
+project=$2
+dir="products/${project}"
+folder=${3:-models}
+
+if [ -z $project ]
+then
+    echo "supply both sqlfluff command (lint/fix) and project folder"
+    exit 1
+fi
+
+if [ -eq "$project" "zoningtaxlots" ]
+then
+    export BUILD_ENGINE_DB="db-ztl"
+else
+    export BUILD_ENGINE_DB="db-$(echo "$project" | tr '_' '-')"
+fi
```

Comment:
this if/else is because our db names aren't 1-to-1 with our data product directory names?

Reply:
Precisely